Widescreen patch
Black Castle - Walkway to throne room
Before Kongol fight cutscene: New start position for Kongol, extended movement
--- original
+++ modified
@@ -351,18 +351,27 @@
 return
 LABEL_23:
 mov 0x7, stor[24]
# New start position and rotation
-gosub inl[:LABEL_7]
+call 101, stor[0], 0xfffffec6, 0xfffffdbc, 0xffffffb0
+call 103, stor[0], 0x0, 0x400, 0x0
+call 106, 0x0
 mov 0x64, stor[24]
 gosub inl[:LABEL_1]
 call 106, 0x0
 call 311, stor[0]
 mov 0x0, stor[24]
 mov 0x8, stor[25]
# Earlier flag trigger for extended walk
-gosub inl[:LABEL_2]
-mov 0x6e, stor[24]
+mov 0x6d, stor[24]
 gosub inl[:LABEL_1]
 call 236, 0x8, 0x7, 0x0, 0x0, 0x0, 0x0
 call 97, 0x1
# Extended walk sequence
+mov 0x8, stor[24]
+gosub inl[:LABEL_24]
+mov 0x7, stor[24]
+gosub inl[:LABEL_24]
# Wait for original walk sequence cue
+call 97, 0x0
+mov 0x6e, stor[24]
+gosub inl[:LABEL_1]
+call 97, 0x1
 mov 0xb, stor[24]
 gosub inl[:LABEL_24]
 mov 0xa, stor[24]
